shared/common-adapters/plain-input.native.tsx

```diff
@@ -230,6 +231,14 @@ class PlainInput extends React.PureComponent<InternalProps> {
       underlineColorAndroid: 'transparent',
     } as const
 
+    if (this.props.allowImagePaste) {
+      common = {
+        ...common,
+        // @ts-ignore, we let this just bubble up
+        onDropped: noop,
```

Comment:
the parent dropview actually consumes this so we don't need to process it here and plumb things down